src/utils/clientToSVGTranslator.ts

```diff
@@ -0,0 +1,71 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module _Util {
+  export class ClientToSVGTranslator {
+    private static _TRANSLATOR_KEY = "__Plottable_ClientToSVGTranslator"
+    private _svg: SVGElement;
+    private _measureRect: SVGElement;
+
+    public static getTranslator(elem: SVGElement): ClientToSVGTranslator {
+      var svg = _Util.DOM.getBoundingSVG(elem);
+
+      var translator: ClientToSVGTranslator = (<any> svg)[ClientToSVGTranslator._TRANSLATOR_KEY];
+      if (translator == null) {
+        translator = new ClientToSVGTranslator(svg);
+        (<any> svg)[ClientToSVGTranslator._TRANSLATOR_KEY] = translator;
+      }
+      return translator;
+    }
+
+    constructor(svg: SVGElement) {
+      this._svg = svg;
+      this._measureRect = <SVGElement> <any>document.createElementNS(svg.namespaceURI, "rect");
```

Comment:
`createElementNS()` supposedly returns an `Element`, even though in this case it returns an `SVGElement`. Can't cast directly from `Element` to `SVGElement`.